commit da33289073de254ab4bacb80b1b83cf9d27c76ea
Author: Lukas Czerner <lczerner@redhat.com>
Date:   Tue Aug 24 14:10:20 2021 +0200

    tests: update expect files for f_large_dir and f_large_dir_csum
    
    Update expect files for f_large_dir and f_large_dir_csum tests to
    include the warning about missing y2038 support with 128-byte inodes.
    
    Fixes: a23b50cd ("mke2fs: warn about missing y2038 support when formatting fresh ext4 fs")
    Signed-off-by: Lukas Czerner <lczerner@redhat.com>
    Signed-off-by: Theodore Ts'o <tytso@mit.edu>

diff --git a/tests/f_large_dir/expect b/tests/f_large_dir/expect
index 028234cc..495ea85d 100644
--- ./tests/f_large_dir/expect
+++ b/tests/f_large_dir/expect
@@ -1,3 +1,4 @@
+128-byte inodes cannot handle dates beyond 2038 and are deprecated
 Creating filesystem with 108341 1k blocks and 65072 inodes
 Superblock backups stored on blocks: 
 	8193, 24577, 40961, 57345, 73729
diff --git a/tests/f_large_dir_csum/expect b/tests/f_large_dir_csum/expect
index aa9f33f1..44770f7b 100644
--- ./tests/f_large_dir_csum/expect
+++ b/tests/f_large_dir_csum/expect
@@ -1,3 +1,4 @@
+128-byte inodes cannot handle dates beyond 2038 and are deprecated
 Creating filesystem with 31002 1k blocks and 64 inodes
 Superblock backups stored on blocks: 
 	8193, 24577
commit 8b2beb24819a976f575e8cec04e3fe6ca8851017
Author: Lukas Czerner <lczerner@redhat.com>
Date:   Tue Aug 24 14:10:19 2021 +0200

    tests: update expect files for f_mmp_garbage
    
    Update expect file for f_mmp_garbage test to work correctly with the
    new default 256 inode size.
    
    Fixes: d730be5ceeba ("tests: update mke2fs.conf to create 256 byte inodes by default")
    Signed-off-by: Lukas Czerner <lczerner@redhat.com>
    Signed-off-by: Theodore Ts'o <tytso@mit.edu>

diff --git a/tests/f_mmp_garbage/expect.1 b/tests/f_mmp_garbage/expect.1
index a8add101..4134eaea 100644
--- ./tests/f_mmp_garbage/expect.1
+++ b/tests/f_mmp_garbage/expect.1
@@ -5,5 +5,5 @@ Pass 2: Checking directory structure
 Pass 3: Checking directory connectivity
 Pass 4: Checking reference counts
 Pass 5: Checking group summary information
-test_filesys: 11/64 files (0.0% non-contiguous), 13/100 blocks
+test_filesys: 11/64 files (0.0% non-contiguous), 15/100 blocks
 Exit status is 0
diff --git a/tests/f_mmp_garbage/expect.2 b/tests/f_mmp_garbage/expect.2
index 66300025..3bca182e 100644
--- ./tests/f_mmp_garbage/expect.2
+++ b/tests/f_mmp_garbage/expect.2
@@ -3,5 +3,5 @@ Pass 2: Checking directory structure
 Pass 3: Checking directory connectivity
 Pass 4: Checking reference counts
 Pass 5: Checking group summary information
-test_filesys: 11/64 files (0.0% non-contiguous), 13/100 blocks
+test_filesys: 11/64 files (0.0% non-contiguous), 15/100 blocks
 Exit status is 0
